### PR TITLE
MAINT: Revisions after previous maintenance commit (#83)

### PR DESCRIPTION
**Pull request by oesteban**
Revise #83 and fix the two outstanding issues:
- [x] Outdated `requirements.txt` file when generating docs.
- [x] Inconsistent handling of the `--output-spaces` section, as a result of copy&paste from fMRIPrep.

### CODE REVIEW

**Comment by oesteban** [comment]
for some obscure reason, circle has stopped to build my PRs :(

**Comment by arokem** [comment]
> for some obscure reason, circle has stopped to build my PRs :(

That's annoying. 

@josephmje : is your PR #81 rebased on current master? Maybe there's a difference between these two PRs that explains why one is tested and the other isn't?

**Comment by josephmje** [comment]
> > for some obscure reason, circle has stopped to build my PRs :(
> 
> That's annoying.
> 
> @josephmje : is your PR #81 rebased on current master? Maybe there's a difference between these two PRs that explains why one is tested and the other isn't?

Yes, I rebased earlier this morning.

**Comment by josephmje** [comment]
With the new version of smriprep, many of the `output_spaces` arguments need to be changed to `spaces`.

**Comment by josephmje** [comment]
I forgot that the documentation used a different set of package versions. That explains why we were still getting nipype errors.

**Comment by oesteban** [comment]
@josephmje have you rebased to this one? should I just merge and check whether I successfully addressed all issues?

File: dmriprep/cli/run.py [hunk 1]
```diff
@@ -416,7 +418,11 @@ def build_workflow(opts, retval):
         retval['return_code'] = 1
         return retval
 
-    output_spaces = parse_spaces(opts)
+    # Initialize --output-spaces if not defined
+    output_spaces = opts.output_spaces
+    if opts.output_spaces is None:
+        from niworkflows.utils.spaces import Reference, SpatialReferences
+        output_spaces = SpatialReferences([Reference("orig", {})])
```
Comment (josephmje) [inline dmriprep/cli/run.py]: "orig" is not a recognized reference

**Comment by josephmje** [comment]
> 
> 
> @josephmje have you rebased to this one? should I just merge and check whether I successfully addressed all issues?

I haven't rebased to this one but I did test locally. The pinned versions are working well. I left a comment about the "orig" space. Also, in the call to `init_anat_preproc_wf()`, the `output_spaces` argument should be changed to `spaces`.

File: dmriprep/cli/run.py (hunk 1 above)
Comment (oesteban) [reply dmriprep/cli/run.py]: shoot yes, will change

Comment (oesteban) [reply dmriprep/cli/run.py]: mmm I'll go with `run` but there's no good option, really: https://github.com/nipreps/niworkflows/blob/master/niworkflows/utils/spaces.py#L8-L17

I'll open an issue for this.

**Comment by oesteban** [comment]
Seems like unfollowing and following from my repo did the trick (regarding to CircleCI).

**Pull request merged**